cmd/nvidia-container-runtime-hook/hook_test.go

```diff
@@ -1,89 +0,0 @@
-package main
-
-import (
-	"encoding/json"
-	"testing"
-
-	"github.com/stretchr/testify/require"
-)
-
-func TestIsPrivileged(t *testing.T) {
```

Comment:
I don't think these tests are covered in the new file `privileged_test.go`. We should readd them.